packages/bits/spec/components/panel/panel.e2e.ts

```diff
@@ -3,150 +3,117 @@ import { ISize } from "selenium-webdriver";
-            expect(await gutter.getResizeDirection()).toEqual("horizontal", "should have correct resize direction");
-        });
-
         it("should make side panel bigger", async () => {
-            const oldCenterPaneSize = await panel.getCenterPaneElementSize();
-            await gutter.moveRight(300);
-            const newCenterPaneSize = await panel.getCenterPaneElementSize();
-            await expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width + 300);
+            const originalSize = await browser.manage().window().getSize();
+            const movePanel = 300;
+            browser.driver.manage().window().setSize(900, 890);
+            const oldCenterPaneSize = await panelResize.getCenterPaneElementSize();
+            await gutter.moveRight(movePanel);
+            const newCenterPaneSize = await panelResize.getCenterPaneElementSize();
+            await expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width + movePanel);
+            // Return to initial state
+            await gutter.moveLeft(movePanel);
+            await browser.manage().window().setSize(originalSize.width, originalSize.height);
         });
 
         it("should make side panel smaller", async () => {
-            const oldCenterPaneSize = await panel.getCenterPaneElementSize();
-            await gutter.moveLeft(100);
-            const newCenterPaneSize = await panel.getCenterPaneElementSize();
-            await expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width - 100);
+            const originalSize = await browser.manage().window().getSize();
+            const movePanel = 20;
+            browser.driver.manage().window().setSize(1200, 880);
+            const oldCenterPaneSize = await panelResize.getCenterPaneElementSize();
+            await gutter.moveLeft(movePanel);
+            const newCenterPaneSize = await panelResize.getCenterPaneElementSize();
+            await expect(oldCenterPaneSize.width).toEqual(newCenterPaneSize.width - movePanel);
+            // Return to initial state
+            await gutter.moveRight(movePanel);
+            await browser.manage().window().setSize(originalSize.width, originalSize.height);
         });
 
         it("should not resize when panel is collapsed", async () => {
-            await panel.toggleExpanded();
-            await expect(await panel.isCollapsed()).toBe(true);
+            await panelResize.toggleExpanded();
+            await expect(await panelResize.isCollapsed()).toBe(true);
 
-            await panel.toggleExpanded();
-            await expect(await panel.isCollapsed()).toBe(false);
-            const oldExpandedCenterPaneSize = await panel.getCenterPaneElementSize();
+            await panelResize.toggleExpanded();
+            await expect(await panelResize.isCollapsed()).toBe(false);
+            const oldExpandedCenterPaneSize = await panelResize.getCenterPaneElementSize();
             await gutter.moveRight(300);
```

Comment:
Maybe capture the `300` in a constant to be reused at line 99 like you've done above?

Reply:
Right, sure.